api/stream/design.go

```diff
@@ -775,6 +812,42 @@ var SinkSettingsPatch = Type("SinkSettingsPatch", func() {
 	Attribute("settings", SettingsPatch)
 })
 
+var GetSinkStatisticsRequest = Type("GetSinkStatisticsRequest", func() {
+	SinkKeyRequest()
+	StatisticsRequest()
+})
+
+var SinkStatisticsResult = Type("SinkStatisticsResult", func() {
+	Attribute("total", Interval)
+	Attribute("intervals", Intervals)
+})
+
+var Intervals = Type("Intervals", ArrayOf(Interval))
+
+var Interval = Type("Interval", func() {
+	Attribute("since", String, func() {
+		Format(FormatDateTime)
+		Example("2022-04-28T14:20:04.000Z")
+	})
+	Attribute("until", String, func() {
+		Format(FormatDateTime)
+		Example("2022-04-28T14:20:04.000Z")
+	})
```

Comment:
Maybe `interval.Start` and `interval.End` is better naming, than `interval.Since`, `interval.Until`?
I don't know, use, what you prefer.